### PR TITLE
chore: make inline docs for events informative

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [X] I branched from develop
- [X] My pull request is against develop
- [X] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [X] I ran `npm run format` and `npm run lint`

### Resolves

Fixes N/A

### Proposed Changes

Updates the inline docs for events to actually be informative/accurate.

### Reason for Changes

Now we can just maintain these instead of also maintaining descriptions on the [events page](https://developers.google.com/blockly/guides/configure/web/events#event_types).

### Test Coverage

No tests! This is just docs changes.

### Documentation

This is documentation silly!

### Additional Information

N/A

### CODE REVIEW

File: core/events/events_comment_move.ts [hunk 1]
```diff
@@ -23,15 +23,20 @@ import type {Workspace} from '../workspace.js';
...
+ * Notifies the developer that a workspace comment has moved.
  *
  * @alias Blockly.Events.CommentMove
  */
 export class CommentMove extends CommentBase {
   override type = eventUtils.COMMENT_MOVE;
+
   comment_?: WorkspaceComment;
+
+  // TODO: We should probably rename these to not have underscores.
```
Comment (cpcallen) [inline core/events/events_comment_move.ts:35]: Can you please open an issue for this TODO, and add the number here?

File: core/events/events_marker_move.ts [hunk 2]
```diff
@@ -24,15 +24,24 @@ import * as eventUtils from './utils.js';
...
+  /** The ID of the block the marker is now on, if any. */
   blockId?: string;
+
+  /** The old node the marker used to be on, if any. */
   oldNode?: ASTNode;
+
+  /** The new node the marker is now on. */
   newNode?: ASTNode;
+
+  /** True if this is a cursor event, false otherwise. */
```
Comment (cpcallen) [inline core/events/events_marker_move.ts:42]: Will the developers understand what a cursor vs. non-cursor marker move is?

File: core/events/events_procedure_parameter_rename.ts [hunk 3]
```diff
@@ -13,13 +13,18 @@ import {ProcedureParameterBase, ProcedureParameterBaseJson} from './events_proce
...
 export class ProcedureParameterRename extends ProcedureParameterBase {
   /** A string used to check the type of the event. */
   type = eventUtils.PROCEDURE_PARAMETER_RENAME;
+
+  /** The new name of the procedure parameter. */
   private readonly newName: string;
 
+  /**
+   * @param oldName The old name of the procedure parameter.
+   */
```
Comment (cpcallen) [inline core/events/events_procedure_parameter_rename.ts:27]:
```suggestion
  /** @param oldName The old name of the procedure parameter. */
```

File: core/events/events_procedure_rename.ts [hunk 4]
```diff
@@ -12,13 +12,17 @@ import {ProcedureBase, ProcedureBaseJson} from './events_procedure_base.js';
 import * as eventUtils from './utils.js';
 
 /**
- * Represents a procedure being renamed.
+ * Notifies the developer that a procedure model has been renamed.
```
Comment (cpcallen) [inline core/events/events_procedure_rename.ts:15]: Again a one-liner, though GitHub won't let me formally suggest the change.

File: core/events/events_procedure_rename.ts [hunk 5]
```diff
@@ -12,13 +12,17 @@ import {ProcedureBase, ProcedureBaseJson} from './events_procedure_base.js';
...
  */
 export class ProcedureRename extends ProcedureBase {
   /** A string used to check the type of the event. */
   type = eventUtils.PROCEDURE_RENAME;
+
   private newName: string;
 
+  /**
+   * @param oldName The old name of the procedure model.
+   */
```
Comment (cpcallen) [inline core/events/events_procedure_rename.ts:25]:
```suggestion
  /** @param oldName The old name of the procedure model. */
```

File: core/events/events_block_move.ts [hunk 6]
```diff
@@ -30,18 +30,46 @@ interface BlockLocation {
...
+
+  /**
+   * The old X and Y workspace coordinates of the block if it was a top level
+   * block. Undefined if it had a parent.
+   */
   oldCoordinate?: Coordinate;
 
+  /**
+   * The ID of the new parent block. Undefined if it is a top-level block.
+   */
```
Comment (cpcallen) [inline core/events/events_block_move.ts:60]:
```suggestion
  /** The ID of the new parent block. Undefined if it is a top-level block. */
```

**Review by cpcallen — APPROVED** [review]
I wonder if the wording "Notifies listeners that…" might be preferable to "Notifies the developer that…", since in most cases the human is not _actually_ being notified.  (Plus there might be developers, plural, and I just don't want to get into that whole can of worms…)

File: core/events/events_block_base.ts [hunk 7]
```diff
@@ -20,12 +20,14 @@ import {Abstract as AbstractEvent, AbstractEventJson} from './events_abstract.js
...
 /**
- * Abstract class for a block event.
+ * Abstract class for any event related to blocks.
  *
  * @alias Blockly.Events.BlockBase
  */
 export class BlockBase extends AbstractEvent {
   override isBlank = true;
+
+  /** The ID of the block associated with this event, or null. */
```
[the remark this answers is not in the document]
Comment (BeksOmega) [reply core/events/events_block_base.ts:30]: blank events -.- but that's a whole can of worms that isn't useful for external developers. So I'll remove that.

File: core/events/events_bubble_open.ts [hunk 8]
```diff
@@ -27,9 +27,15 @@ import type {Workspace} from '../workspace.js';
  * @alias Blockly.Events.BubbleOpen
  */
 export class BubbleOpen extends UiBase {
+  /** The ID of the block the bubble is attached to. */
   blockId?: string;
+
+  /** True if the bubble is opening, false otherwise. */
```
[the remark this answers is not in the document]
Comment (BeksOmega) [reply core/events/events_bubble_open.ts]: Yep false if it is closing! Updated text.

**Pull request merged**